tests/src/runtimeApi/memory/hipMemcpy.cpp

```diff
@@ -33,6 +33,8 @@ THE SOFTWARE.
 #include "hip/hip_runtime.h"
 #include "hip/hip_runtime.h"
 #include "test_common.h"
+#include "sys/types.h"
```

Comment:
this does not work on windows. you may check https://stackoverflow.com/questions/2513505/how-to-get-available-memory-c-g

Reply:
Which macro do you use to guard linux and windows?

Reply:
#if _WIN32